extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ColumnPageReaderImpl.java

```diff
@@ -149,23 +153,35 @@ private DataPageV1 readV1Unsafe(
         }
         final int uncompressedPageSize = pageHeader.getUncompressed_page_size();
         final int compressedPageSize = pageHeader.getCompressed_page_size();
-        final BytesInput decompressedInput =
-                compressorAdapter.decompress(in, compressedPageSize, uncompressedPageSize, channelContext);
-        final DataPageHeader header = pageHeader.getData_page_header();
-        return new DataPageV1(
-                decompressedInput,
-                header.getNum_values(),
-                uncompressedPageSize,
-                null, // TODO in the future might want to pull in statistics
-                getEncoding(header.getRepetition_level_encoding()),
-                getEncoding(header.getDefinition_level_encoding()),
-                getEncoding(header.getEncoding()));
+        return compressorAdapter.decompress(in, compressedPageSize, uncompressedPageSize, channelContext);
+    }
+
+    /**
+     * Helper struct for holding the fields of a DATA_PAGE_V2 page while reading it.
+     */
+    // TODO Better name for this class
```

Comment:
TODO

Reply:
Helper -> Partial, or Helper to Continuation?